### PR TITLE
build/migrate to next10

### PR DESCRIPTION
**Pull request by new5558**
#66

### CODE REVIEW

File: pages/_document.jsx [hunk 1]
```diff
@@ -9,21 +9,7 @@ class MyDocument extends Document {
...
-          />
-          <meta
-            name="keywords"
-            content="Chula Pop Bus, Thinc, วิศวะคอมพ์จุฬา, ise chula, chulalongkorn, วิศวะจุฬา, open registration, cu blood"
-          />
-          <meta httpEquiv="content-language" content="th,en" />
-          <meta name="robots" content="index, follow" />
-          <meta httpEquiv="Content-Type" content="text/html; charset=utf-8" />
-        </Head>
+        <Head></Head>
```
Comment (saenyakorn) [inline pages/_document.jsx]: why empty `Head` na?

Comment (new5558) [reply pages/_document.jsx]: Head moved to app.js per nextjs error
https://nextjs.org/docs/messages/no-document-viewport-meta

Comment (saenyakorn) [reply pages/_document.jsx]: I know that `Head` need to be moved to `_app.js`. But why we need empty `Head` in `_document.js` ?

Comment (new5558) [reply pages/_document.jsx]: Thanks for explanation. I just fixed and changed to `<Head />` per this document https://nextjs.org/docs/messages/missing-document-component

**Review by saenyakorn — APPROVED** [review]

**Pull request merged**